packages/web-components/src/components/image/__stories__/image.stories.ts

```diff
@@ -0,0 +1,49 @@
+/**
+ * @license
+ *
+ * Copyright IBM Corp. 2020
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import { html } from 'lit-element';
+// import Add16 from 'carbon-custom-elements/es/icons/add/16';
```

Comment:
Did you still want to add icon example, or do you want to remove this line?

Reply:
Good catch, i'm gonna remove this